source/develop/release-management/features/ovirt-ovn-provider.md

```diff
@@ -276,6 +276,76 @@ modifies the local OpenFlows on the host and deletes the chassis id from the
+
+The oVirt engine manages the provider's identity by storing the provider's
+certificate in the external providers trust store and check the provider's
+certificate during connecting to the provider.
+
+## Authentication and Authorization at the provider
+
+The oVirt OVN provider is able to authenticate users and authorize requests
+according a minimal subset of the [OpenStack API](https://developer.openstack.org/api-guide/quick-start/).
+The procedure is that the client [authenticate](#authentication) itself to get
+a token, which has to be used in the [subsequent requests](#ovirt-ovn-provider-1).
+
+The behavior of authentication and authorization is defined by the
+[plugin](#plugins) chosen in the provider's configuration.
+
+### Authentication
+
+Authentication is implemented by a single request of the OpenStack
+[Identity API v2.0](https://developer.openstack.org/api-ref/identity/v2/),
+commonly known as keystone API.
+
+The provider handles the following request for authentication:
+
+*   POST tokens - authenticates and generates a token from username and
+    password, using a subset of the parameters of the OpenStack
+    [specification](https://developer.openstack.org/api-ref/identity/v2/?expanded=authenticate-detail#authenticate).
+
+### Authorization
+
+The `X-Auth-Token` HTTP header in every request to the OpenStack Rest API is
+used to authorize the request. If the authorization is given, is up to the
+configured plugin of the provider.
+
+### Plugins
+
+There are various plugins available, which differentiate by their behavior:
+
+*  NoAuthPlugin - generates a static token, independently from the provided
+   username and password, and accepts every request to the OpenStack Rest API.
+   Even requests, which does not contain the `X-Auth-Token` HTTP header are
+   accepted.
+
+*  MagicTokenPlugin - generates a static token, independently from the provided
+   username and password. Request to the OpenStack Rest API must provide this
+   static token.
+
+*  AuthorizationByUserName - uses the oVirt engine's
+   [SSO](/develop/release-management/features/infra/uniformssosupport/) to
+   create a token from the given username and password. So the user has to be a
+   valid user of oVirt engine. The token in the requests to the OpenStack Rest
+   API is authorized, if it is associated to the username defined in the
+   provider's configuration.
+
+*  AuthorizationByRole - uses the oVirt engine's SSO to
+   create a token from the given username and password. The token in the
+   requests to the OpenStack Rest API is authorized, if it is associated to
+   a user with the role defined in the provider's configuration.
+
+*  AuthorizationByGroup - uses the oVirt engine's SSO to
+   create a token from the given username and password. The token in the
+   requests to the OpenStack Rest API is authorized, if it is associated to
+   a user which is a member of the group defined in the provider's configuration.
```

Comment:
It's great to have a pluggable design, but I would not like to support all these plugins. By default, AuthorizationByGroup is to be used, and we do not intend to support any other.
Please state so clearly in this page. Also specify the default trusted group, and how a use can change it, such as in the case where the admin user is a member of an ldap group.